src/Illuminate/Support/Str.php

```diff
@@ -322,6 +322,8 @@ public static function snake($value, $delimiter = '_')
 			$value = strtolower(preg_replace('/(.)(?=[A-Z])/', '$1'.$delimiter, $value));
 		}
 
+		$value = preg_replace("/([ { $delimiter }]+)/", $delimiter, $value);
```

Comment:
The curly braces usually hug the variable throughout the codebase: `"/([ {$delimiter}]+)/"`